app/uk/gov/hmrc/hmrcemailrenderer/templates/ats/annualTaxSummariesMessageAlert_cy.scala.txt

```diff
@@ -16,6 +16,6 @@ Mae crynodebau treth er gwybodaeth yn unig, felly nid oes angen cymryd unrhyw ga
 
 Myrtle Lloyd
 
-Cyfarwyddwr Cyffredinol Gwasanaethau i Gwsmeriaid CThEF
+Cyfarwyddwr Cyffredinol Gwasanaethau i Gwsmeriaid CThEFÏ
```

Comment:
Is this change expected? I believe this ticket is only for tax year changes in other file

Reply:
I think I did it by mistake, I have corrected it and created on another commit. Please have a look.